File: Android/samples/graphnotificationssample/app/src/main/java/com/microsoft/connecteddevices/graphnotifications/Secrets.java
//
// Copyright (c) Microsoft Corporation. All rights reserved.
//

package com.microsoft.connecteddevices.graphnotifications;

class Secrets {
// These come from the converged app registration portal at apps.dev.microsoft.com
    // MSA_CLIENT_ID:           Id of this app's registration in the MSA portal
    // AAD_CLIENT_ID:           Id of this app's registration in the Azure portal
    // AAD_REDIRECT_URI:        A Uri that this app is registered with in the Azure portal.
    //                          AAD is supposed to use this Uri to call the app back after login (currently not true, external requirement)
    //                          And this app is supposed to be able to handle this Uri (currently not true)
    // APP_HOST_NAME            Cross-device domain of this app's registration
    static final String MSA_CLIENT_ID = "<<MSA client ID goes here>>";
    static final String AAD_CLIENT_ID = "<<AAD client ID goes here>>";
    static final String AAD_REDIRECT_URI = "<<AAD redirect URI goes here>>";
    static final String APP_HOST_NAME = "<<App cross-device domain goes here>>";

    // Your client's Firebase Cloud Messaging Sender Id
    static final String FCM_SENDER_ID = "<<FCM sender ID goes here>>";
}

File: Windows/samples/GraphNotificationsSample/Secrets.cs
namespace SDKTemplate
{
    public class Secrets
    {
        // Get the following from Application Registration Portal (apps.dev.microsoft.com)
        // MSA_CLIENT_ID:           Application Id for MSA
        // AAD_CLIENT_ID:           Application Id for AAD
        // AAD_REDIRECT_URI:        Redirect Uri for AAD
        // APP_HOST_NAME            Cross-device domain for UserDataFeed

        // These values are specific to this app, and can't be used by your app. You will need to register
        // your app and get your own secrets, these are ours.
        public static readonly string MSA_CLIENT_ID = "<<MSA client ID goes here>>";
        public static readonly string AAD_CLIENT_ID = "<<AAD client ID goes here>>";
        public static readonly string AAD_REDIRECT_URI = "<<AAD redirect URI goes here>>";
        public static readonly string APP_HOST_NAME = "<<App cross-device domain goes here>>";
    }
}
